frontend/apps/design-system-storybook/package.json

```diff
@@ -56,17 +56,21 @@
     "@types/react": "^18.3.12",
     "axe-playwright": "^2.0.3",
     "concurrently": "^9.1.0",
+    "css-loader": "^7.0.0",
+    "dedent": "^1.7.0",
```

Comment:
pnpm forces packages to be listed instead of inferred by package.json's outside of the package.